### PR TITLE
Bring your own tsdb

### PR DESCRIPTION
**Pull request by jessegoodier**

### CODE REVIEW

File: azure/README-enterprise.md [hunk 1]
```diff
@@ -30,10 +30,13 @@ kubectl create secret generic kubecost-thanos -n kubecost --from-file=object-sto
 # Create Cloud Integration Secret
 kubectl create secret generic cloud-integration -n kubecost --from-file=cloud-integration.json
 
-# Azure service key
+# Azure service key (optional)
```
Comment (kc-adawson) [inline azure/README-enterprise.md:33]: Quick comment about what the service key option turns on/what it limits?

Comment (jessegoodier) [reply azure/README-enterprise.md:33]: finding unused LBs and PVs from my understanding. also spot prices in azure?

Comment (jessegoodier) [reply azure/README-enterprise.md:33]: I can add that detail there, it is just spread out a bit

**Review by kc-adawson — APPROVED** [review]
LGTM

**Pull request merged**